fastadmin/models/helpers.py

```diff
@@ -15,6 +15,21 @@
     ModelSchema,
 )
 
+try:
```

Comment:
Could we remove this code and use the following

class Parent:
    pass

class Child(Parent):
    pass

print(Child.__bases__[0].__name__)  # Prints the parent class name

and check strings

Reply:
@artrey

Reply:
probably admin classes can have mixins so we need to check not only [0]

Reply:
Agreed with you. Ok let's revert previous changes (sorry for that) and we will merge

Reply:
@artrey